hw/lint/master_lint_cfgs_list.hjson

```diff
@@ -0,0 +1,34 @@
+// Copyright lowRISC contributors.
```

Comment:
Hi Michael, thanks for the implementation. This definitely helps with the FPV flow. I wonder if you consider making the structure of the folder like dv instead of flattening out every file? I think under DV we have `data`, `doc`, and `tools`.